AzureCosmosDB/csharp/DocumentVectorPipelineFunctions/BlobTriggerFunction.cs

```diff
@@ -60,21 +63,41 @@ private async Task HandleBlobCreateEventAsync(BlobClient blobClient)
         var maxTokensPerChunk = configuration.GetValue<int>(MaxTokensPerChunkName, DocumentChunker.DefaultMaxTokensPerChunk);
         var overlapTokens = configuration.GetValue<int>(OverlapTokensName, DocumentChunker.DefaultOverlapTokens);
 
-        this._logger.LogInformation("Analyzing document using DocumentAnalyzerService from blobUri: '{blobUri}' using layout: {layout}", blobClient.Name, "prebuilt-read");
+        var extension = Path.GetExtension(blobClient.Name);
+        var textChunks = new List<TextChunk>();
+        if (extension == ".txt")
+        {
+            using var stream = await blobClient.OpenReadAsync();
+            var lines = await ReadAllLinesAsync(stream);
+            textChunks.AddRange(DocumentChunker.ChunkTextLines(
+                lines, maxTokensPerChunk, overlapTokens));
+        }
+        else if (extension == ".md")
```

Comment:
are the result for markdown files are pretty similar to DI? Can i handle all tags/images/tables on markdown files and extract text content correctly or ignore them so that the extract text chunks have readable information?

Reply:
> are the result for markdown files are pretty similar to DI?

Document Intelligence doesn't handle Markdown content today, so this is purely additive.

>  Can i handle all tags/images/tables on markdown files and extract text content correctly or ignore them so that the extract text chunks have readable information?

Markdown is fundamentally a text based format - there are no "images" embedded in it.  Instead, it's like html where there is a reference to an external image.  This will *not* interpret the content of those images.  Tables should be handled, but perhaps not sturcturally recognized.